### PR TITLE
add links about precompiled binaries in FAQ and how-to-add

### PR DESCRIPTION
**Pull request by memsharded**

### CODE REVIEW

File: docs/how_to_add_packages.md [hunk 1]
```diff
@@ -164,6 +164,8 @@ For C/C++ projects which use CMake for building, you can take a look on [szip](h
 
 However, if you need to use autotools for building, you can take a look on [mpc](https://github.com/conan-io/conan-center-index/blob/master/recipes/mpc/all/conanfile.py), [libatomic_ops](https://github.com/conan-io/conan-center-index/blob/master/recipes/libatomic_ops/all/conanfile.py), [libev](https://github.com/conan-io/conan-center-index/blob/master/recipes/libev/all/conanfile.py).
 
+Please be aware that ConanCenter only build from sources and [pre-compiled binaries are not acceptable](https://github.com/conan-io/conan-center-index/blob/master/docs/packaging_policy.md)
```
Comment (prince-chrismc) [inline docs/how_to_add_packages.md:167]:
```suggestion
Please be aware that ConanCenter only builds from sources and [pre-compiled binaries are not acceptable](https://github.com/conan-io/conan-center-index/blob/master/docs/packaging_policy.md)
```

File: docs/faqs.md [hunk 2]
```diff
@@ -29,6 +29,12 @@ However, if it is not possible and there is the case of a new recipe producing a
 
 For example, `GSL` is the name of `Guidelines Support Library` from Microsoft and `GNU Scientific Library` from GNU. Both libraries are commonly known as `gsl`, however, to disambiguate (if there is already a `gsl` package in this repo) we could use `ms-gsl` in the first case or `gnu-gsl` in the second.
 
+## What is the policy on creating packages from pre-compiled binaries?
+
+The policy is that in the general case [recipes should build packages from sources](https://github.com/conan-io/conan-center-index/blob/master/docs/packaging_policy.md), because of reproducibility and security concerns. The implication is that the sources must be publicly available, and in a format that can be consumed programmatically.
+
+Check the link for further details.
```
Comment (prince-chrismc) [inline docs/faqs.md:36]: 🤔 What link?

Comment (jgsogo) [reply docs/faqs.md:36]: Link in the previous paragraph. I think it is clear enough, isn't it?

**Review by jgsogo — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

File: docs/faqs.md (hunk 2 above)
Comment (SSE4) [reply docs/faqs.md:36]: well, it's more clear if link would be a hyperlink itself, but never mind

**Review by SSE4 — APPROVED** [review]

**Pull request merged**